charts/timed/Chart.yaml

```diff
@@ -15,8 +15,3 @@ maintainers:
   - name: adfinis-sygroup
     email: support@adfinis.com
     url: https://adfinis.com
-dependencies:
-  - name: postgresql
-    version: ~9.1.2
-    repository: https://charts.bitnami.com/bitnami
-    condition: postgresql.enabled
```

Comment:
you will probably still need the dependecy just so there is a database on ci, otherwise chart-testing might fail the install bits